easy_rec/python/model/deepfm.py

```diff
@@ -94,3 +95,12 @@ def build_predict_graph(self):
     self._add_to_prediction_dict(output)
 
     return self._prediction_dict
+
+  def build_feature_output_dict(self):
+    outputs = super(DeepFM, self).build_output_dict()
```

Comment:
should it be super(DeepFM, self).build_feature_output_dict()?

Reply:
fixed